hack/lib/catalogsource.bash

```diff
@@ -85,9 +85,20 @@ spec:
           /bin/opm registry serve -d index.db -p 50051
 EOF
 
-  # Wait for the index pod to be up to avoid inconsistencies with the catalog source.
-  wait_until_pods_running "$OLM_NAMESPACE"
```

Comment:
Do we still need to wait somewhere for the CatalogSource itself to become ready? The pod takes quite a while to come up. Maybe it'd be a compromise to wait for the Available condition on the deployment.